third_party/move/move-stdlib/src/natives/debug.rs

```diff
@@ -1,652 +0,0 @@
-    #[cfg(feature = "testing")]
-    {
-        let canonical = false;
-        let single_line = false;
-        let include_int_types = false;
-
-        let mut out = "[debug] ".to_string();
-        let val = _val.value_as::<Reference>()?.read_ref()?;
-
-        testing::print_value(
-            _context,
-            &mut out,
-            val,
-            _ty,
-            &_move_std_addr,
-            0,
-            canonical,
-            single_line,
-            include_int_types,
-        )?;
-        println!("{}", out);
-    }
-
-    Ok(NativeResult::ok(gas_params.base_cost, smallvec![]))
-}
-
-pub fn make_native_print(
-    gas_params: PrintGasParameters,
-    move_std_addr: AccountAddress,
-) -> NativeFunction {
-    Arc::new(
-        move |context, ty_args, args| -> PartialVMResult<NativeResult> {
-            native_print(&gas_params, context, ty_args, args, move_std_addr)
-        },
-    )
-}
-
-/***************************************************************************************************
- * native fun print_stack_trace
- *
- *   gas cost: base_cost
- **************************************************************************************************/
-#[derive(Debug, Clone)]
-pub struct PrintStackTraceGasParameters {
-    pub base_cost: InternalGas,
-}
-
-#[allow(unused_variables)]
-#[inline]
-fn native_print_stack_trace(
-    gas_params: &PrintStackTraceGasParameters,
-    context: &mut NativeContext,
-    ty_args: Vec<Type>,
-    args: VecDeque<Value>,
-) -> PartialVMResult<NativeResult> {
-    debug_assert!(ty_args.is_empty());
-    debug_assert!(args.is_empty());
-
-    #[cfg(feature = "testing")]
-    {
-        let mut s = String::new();
-        context.print_stack_trace(&mut s)?;
-        println!("{}", s);
-    }
-
-    Ok(NativeResult::ok(gas_params.base_cost, smallvec![]))
-}
-
-pub fn make_native_print_stack_trace(gas_params: PrintStackTraceGasParameters) -> NativeFunction {
-    Arc::new(
-        move |context, ty_args, args| -> PartialVMResult<NativeResult> {
-            native_print_stack_trace(&gas_params, context, ty_args, args)
-        },
-    )
-}
-
-/***************************************************************************************************
- * module
- **************************************************************************************************/
-#[derive(Debug, Clone)]
-pub struct GasParameters {
-    pub print: PrintGasParameters,
-    pub print_stack_trace: PrintStackTraceGasParameters,
-}
-
-pub fn make_all(
-    gas_params: GasParameters,
-    move_std_addr: AccountAddress,
-) -> impl Iterator<Item = (String, NativeFunction)> {
-    let natives = [
-        ("print", make_native_print(gas_params.print, move_std_addr)),
-        (
-            "print_stack_trace",
-            make_native_print_stack_trace(gas_params.print_stack_trace),
-        ),
-    ];
-
-    make_module_natives(natives)
-}
-
-#[cfg(feature = "testing")]
-mod testing {
```

Comment:
Was wondering if we should remove those debug natives. They could be used by some external folks?

Reply:
Can't external folks use aptos debug natives? Do we know if they use them? If not I would prefer to get rid of these and shift our debug implementation to Move repo for example :)

Reply:
cc @jolestar

Reply:
We use `std::debug`, and many examples docs use `std::debug`. I suggest we keep the `std::debug` or shift `aptos_std::debug` to `std::debug`.

Reply:
Good to know! Thanks!

Reply:
@georgemitenkov I don't think external folks can use the latest version of Aptos natives easily -- those are now dependent on the our gas schedule.

Reply:
Makes sense, closing the PR and let's plan more incremental changes in this case.